core/server/models/plugins/filter.js

```diff
@@ -61,6 +61,7 @@ const filter = function filter(Bookshelf) {
             let extra = this.extraFilters(options);
             let overrides = this.enforcedFilters(options);
             let defaults = this.defaultFilters(options);
+            let transformer = options.mongoTransformer;
```

Comment:
Was thinking about the naming of `mongoTransformer` a little and not sure if we want to be specific about "mongo" bit here. For reference to other fields, we have "filter" named as a "filter", not `nqlFilter`, or `defaultFilters` not `defaultNQLFilters` etc. 

In a similar manner would lean towards naming this variable as `filterTransformer`. Wdyt @allouis 	?

Reply:
Hmmm :thinking: I think `filterTransformer` would imply that the transformer acts on filters defined in the NQL Language (ATM Machine, PIN Number, I know :sweat_smile:) but this works on the Mongo query definition.

I _did_ think that mongo was too much "internals" but the [NQL](https://github.com/NexesJS/NQL) README explicitly states that it is for parsing NQL filters to mongo queries - so I'm not so against it